Assignment_5/Assignment_5_Problem_4.js

```diff
@@ -0,0 +1,14 @@
+//Assignment 5
+// Problem 4
+// Prolem statement 
+// http://www.codewars.com/kata/santaclausable-interface
+
+// Solution
+
+function isSantaClausable(obj) {
+    return (
+        (typeof obj.sayHoHoHo === 'function') &&
```

Comment:
can extract 'function' to another const variable and use the variable everywhere instead of string